src/metpy/constants/default.py

```diff
@@ -37,9 +37,11 @@
     Cp_l = water_specific_heat = units.Quantity(4.2194, 'kJ / kg / K').to('J / kg / K')
     Lv = water_heat_vaporization = units.Quantity(2.50084e6, 'J / kg')
     Lf = water_heat_fusion = units.Quantity(3.337e5, 'J / kg')
+    Ls = water_heat_sublimation = Lv + Lf
     Cp_i = ice_specific_heat = units.Quantity(2090, 'J / kg / K')
     rho_i = density_ice = units.Quantity(917, 'kg / m^3')
     sat_pressure_0c = units.Quantity(6.112, 'millibar')
+    T0 = water_triple_point_temperature = units.Quantity(273.16, 'K')
```

Comment:
Was $$T_0$$ what was used in the cited paper?

Reply:
![image](https://github.com/user-attachments/assets/894dae7f-923d-4258-9034-99829715e695)
If your asking re variable name, I had it as `T_0` before but changed to mirror `P0`. Doesn't really matter to me either way.

Reply:
Sounds good.